common/messages/fullServerFault.go

```diff
@@ -413,6 +413,24 @@ func (m *FullServerFault) String() string {
 		m.Timestamp.GetTimeSeconds())
 }
 
+func (m *FullServerFault) StringWithSigCnt(s interfaces.IState) string {
+	if m == nil {
+		return "-nil-"
+	}
+	return fmt.Sprintf("%6s-vm%02d[%d] (%v) AuditID: %v DBHt:%5d SysHt:%3d Clr:%t -- hash[:3]=%x Valid Sigs: %d TS:%d",
```

Comment:
I think the proper go style for using multiple lines in function arguments is more like:
```
fmt.Sprint(
	"string",
	arg2,
	arg3,
)
```